demos/Web View/main.py

```diff
@@ -0,0 +1,86 @@
+import gi
+
+gi.require_version("WebKit", "6.0")
+from gi.repository import GLib, GObject, WebKit
+import workbench
+
+dummy = WebKit.WebView()  # Throw away object to make Gtk.Builder aware of this type
```

Comment:
Hi! You can remove this now, we added a `type_ensure` to the previewer. On current Workbench nightly your demo should now work without. Let me know if you run into any issues!

Reply:
Actually, sorry, it's not merged yet! You can test based on this PR: https://github.com/workbenchdev/Workbench/pull/879